deploy/csv/annotations.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
We did away entirely with the `/deploy` directory since it was confusing; the entire repository contains different deployment configurations and the `/deploy` directory simply held one such configuration for the operator. This directory doesn't exist any more and I think that the OLM stuff should either be nested in the `/operator` directory or in a new top level directory called `/olm`